### PR TITLE
Update some "Coming Soon" sections for DR9, and some general clean-up.

### PR DESCRIPTION
**Pull request by geordie666**
This PR adds some quick but needed fixes and updates for the DR9 pages:
- Remove the "Coming Soon" language for the `photo-z` files and `masking` directory, now that we've made those public.
- Fix some broken links for the cutout server.
- Add some BASS collaborators that were missing on the `Contacts` page.

I'm going to leave this as WIP for now, but I thought I'd push these updates in case we decide we want to serve a new version of the website sooner rather than later.

### CODE REVIEW

**Comment by weaverba137** [comment]
I've researched the technique for renaming the master branch to main, as suggested in #132, but I suggest we do that *after* this PR is merged.

Is this PR ready for review?  Do we need to deploy the changes from #132 *first*, then review, merge and deploy this PR?  Or can we incorporate both PRs into a single deployment?

**Comment by geordie666** [comment]
@weaverba137: Feel free to (review and/or) merge this PR if you'd like. It is complete and self-contained. 

I was just leaving this open in case I found time to have another general read-through of the whole website before you deploy (this and) the new work on the SGA. I can always pick up new changes in a different PR, though. We shouldn't deploy #132 until we've also merged these changes, unless you're happy to deploy the website twice in quick succession.

File: CHANGES.md
Comment (weaverba137) [inline CHANGES.md:3]: I suggest calling this 9.1.0 since the SGA PR was a fairly significant change.

**Review by weaverba137 — APPROVED** [review]
Looks fine to me, only one minor comment and it's only on the change log.

File: CHANGES.md
Comment (geordie666) [reply CHANGES.md:3]: I agree with calling this `9.1.0`: But, I assume you'll make that change whenever you cut a new tag? Particularly if @moustakas might make additional changes in a different PR before we deploy?

Comment (weaverba137) [reply CHANGES.md:3]: Sure.  Just noting that as a reminder.

**Comment by weaverba137** [comment]
@geordie666, how soon do you need this deployed?  Monday would be somewhat easier for me.

**Comment by geordie666** [comment]
@weaverba137: There's no need to deploy this, yet. I think we're still waiting on some SGA updates from @moustakas? I merged this PR as I started another branch to make a few further updates for DR9. I'll push that new branch later today or early tomorrow.

Before we serve a new version of the website, we probably want to copy the forced photometry files into place (the location of which @dstndstn circulated to decam-chatter by email on March 9th), and let @moustakas finalize the SGA.

**Comment by geordie666** [comment]
Also, thanks for deleting the branch. Can't believe I forgot to do that.

**Pull request merged**
